### DISCUSSION TITLE
storage with different input and output

### DISCUSSION THREAD

**Discussion by mistervulf** [SESMG/SESMG#116] [Q&A]
Wie modelliere ich eine Solarthermieanlage, welche über einen Wärmetauscher in einen Speicher die Wärme abgibt?
Wenn ich eine Solarthermie modelliere und einen thermischen Speicher wird dieser übergangen. 
![image](https://user-images.githubusercontent.com/93590182/178700251-9af73ef6-4699-4da8-b179-d4d924688d11.png)

**Comment by GregorBecker** [comment]
With the current state of the repository it is not possible to map this, we can gladly reprogram it for your application context. You would have to create a pull request from your forked repository to the master branch. Then I'll take a look at it and implement it. 
Kind regards,
Gregor
